gitversioner/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'kotlin'
+apply plugin: 'com.novoda.bintray-release'
+
+group = 'com.pascalwelsch.gitversioner'
+version = '0.0.1-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.1.2-4'
```

Comment:
Why declare it inside a buildscript and add it to the `ExtensionContainer`? 🤔 
I think it is easier to read to put it out of the buildscript (directly under `version`) and use it like it is :)